luxonis_train/models/luxonis_model.py

```diff
@@ -259,8 +262,8 @@ def forward(
 
         @type inputs: L{Tensor}
         @param inputs: Input tensor.
-        @type labels: L{Labels} | None
-        @param labels: Labels dictionary. Defaults to C{None}.
+        @type task_labels: L{Labels} | None
```

Comment:
```suggestion
        @type task_labels: L{TaskLabels} | None
```
I think this type just needs an update